src/Aspire.Hosting.Azure/build/Aspire.Hosting.Azure.targets

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
I think we only need a .targets file in `buildTransitive`. We shouldn't need the other 2 targets files. Just move this logic in there.

Reply:
sure, removed all unnecessary ones